Dockerfile

```diff
@@ -13,6 +13,7 @@ FROM ubuntu:24.04 AS production
 COPY --from=build /app/.pixi/envs/default /app/.pixi/envs/default
 COPY --from=build /app/app.py /app
 COPY --from=build /app/app_config.toml /app
+COPY --from=build /app/version_info.json /app
```

Comment:
just as a side note: `COPY`ing `.git` results in the docker layer caching not properly working. not that this really matters here

Reply:
Do we copy `.git`? 🤔

Reply:
otherwise `pixi run -e default save-version-info` would not work, right?

we do it implicitly by running `COPY . /app` and not excluding `.git` in `.dockerignore`

Reply:
ahh okay I think I missed that you were referring to the build container; `.git` does not end up in production ofc

Reply:
then you're right, I think we should not care about this